quasar-core/src/test/java/co/paralleluniverse/fibers/instrument/MethodDatabaseDuplicate.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2008-2013, Matthias Mann
```

Comment:
Who is this guy? Did he write this test?

Reply:
Yes, spotted the comment post commit will remove, one of the delights of folding editors ;)